packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -198,6 +198,11 @@ export type CallWithChatCompositeOptions = {
      * @defaultValue false
      */
     hideSurvey?: boolean;
+    /* @conditional-compile-remove(end-of-call-survey-self-host) */
+    /**
+     * Optional callback to add extra logic when survey is dismissed. For self-host only
+     */
+    onSurveyDismissed?: () => void;
```

Comment:
This feels like something that Contoso might want if we are wanting it in our devlopment. would be interested to see what they have to say in the devex if this is included 😊

Reply:
ARB for this feature is already completed, we can save this for version 2